setup.py

```diff
@@ -38,80 +38,85 @@
     keywords='Magnetic Resonance Imaging MRI spinal cord analysis template',
     packages=find_packages(exclude=['.git', 'data', 'dev', 'dev.*',
                                     'install', 'testing']),
+    data_files=[
+        # <hack>
+        ("sct_scripts", [os.path.join("scripts", x) for x in os.listdir("scripts") if x.endswith(".py")]),
+        # </hack>
+    ],
```

Comment:
Was this on purpose? It's in b10c820c809087794536e9fe88b61b6eb443522f which doesn't explain it in its commit message.

I would expect `console_scripts` handles making sure these get included in the right places.

Reply:
This was recently removed, in #2837, but it looks like it snuck back in.

Reply:
🤭my bad

Reply:
That's what code review is for!